libethcore/BlockHeader.cpp

```diff
@@ -116,8 +116,10 @@ h256 BlockHeader::hash(IncludeSeal _i) const
 
 void BlockHeader::streamRLPFields(RLPStream& _s) const
 {
+    bigint nnn = m_number;     // Fix rlp stream issue with uint65_t
```

Comment:
Also there are some proposals to require the limit for block nuumber and timestamp, so I'm not sure it makes sense to change it now.
https://eips.ethereum.org/EIPS/eip-1985

Reply:
It fixes the test. Because RLPStream << was working incorrectly with u64int_t

Reply:
What do you mean exactly? Maybe we should fix `operator<<` for `uint64_t` or something

(Some tests currently fail anyway)

Reply:
Yes. I had blockchain/invalidblocks/bcInvalidHeader/timestamp4 test failing without this change. I assume the issue is in << operation

Reply:
I can't find this test, what are header values there?

Reply:
sorry I was out of laptop.
it is this test: 
`BlockchainTests/InvalidBlocks/bcUncleHeaderValidity/incorrectUncleTimestamp4`

I set timestamp of the uncleheader to 500000000000 + previous block (~1000)
and the value I get `rlp::85746a528800`.  but testeth gives me RLP encode of this argument `rlp::846a528800`
unless I explicitly say that it is a bigint in that function. the fix is dirty because I just try to make the tests pass